moped-database/migrations/1746640451690_run_sql_migration/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Ensures that projects can only have one current phase
+CREATE UNIQUE INDEX moped_proj_phases_partial_constraint ON moped_proj_phases (project_id)
+    WHERE (is_current_phase = TRUE AND is_deleted = FALSE);
+    
```

Comment:
it thinks there isnt a blank line at the end of the file because this line has empty spaces in it. I ran it through a linter and it removed the blank lines